digitalpalireader/content/js/web/navigation_sidebar.js

```diff
@@ -308,26 +308,14 @@ var DPRNav = {
   },
 
   gotoPlace: function ([nikaya, book, meta, volume, vagga, sutta, section, hiert]) {
-    document.getElementById('set').value = nikaya;
-    for (i in G_hshort) {
-      if (i == hiert)
-        document.getElementById(G_hshort[i]).setAttribute('checked', true);
-      else
-        document.getElementById(G_hshort[i]).checked = false;
-    }
-    this.changeSet(1, book);
-    this.switchhier(hiert);
-    document.getElementById('book').selectedIndex = book;
-    DPRXML.updateHierarchy(0);
-    document.getElementById('meta').selectedIndex = meta;
-    DPRXML.updateHierarchy(1);
-    document.getElementById('volume').selectedIndex = volume;
-    DPRXML.updateHierarchy(2);
-    document.getElementById('vagga').selectedIndex = vagga;
-    DPRXML.updateHierarchy(3);
-    document.getElementById('sutta').selectedIndex = sutta;
-    DPRXML.updateHierarchy(4);
-    document.getElementById('section').selectedIndex = section;
+    document.getElementById('nav-set').value = nikaya;
```

Comment:
please change this back to what it was. i.e. lets not use click(). 
new elementnames are fine (i.e. 'book' -> 'nav-book')

the issue with more than one of M/A/T getting selected can be handled down the line.